server/config/config_test.go

```diff
@@ -568,6 +577,9 @@ func TestMicrosoftWSTEPConfig(t *testing.T) {
 	}
 }
 
+func TestS3Config(t *testing.T) {
+}
```

Comment:
To remove or implement?

Reply:
Removing as the test at the top was updated to handle new config